### PR TITLE
Added embedded video to markdown-it preview.

### PR DESCRIPTION
**Pull request by bharney0**

### CODE REVIEW

File: docs-preview/src/markdown-extensions/video.ts [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+    marker: ">",
+    validate(params) {
+        return params.trim().match(regex);
+    },
+    render(tokens, idx) {
+        // const matches = tokens[idx].info.trim().match(regex);
+        const VIDEO_RE = /^\[\!VIDEO\s+(.+)\]$/gmi;
+        const videoMatches = VIDEO_RE.exec(tokens[idx].info.trim());
+        if (videoMatches !== null) {
+            return `<iframe src="${videoMatches[1]}" width="640" height="320" allowFullScreen="true" frameBorder="0"></iframe>`;
```
Comment (bharney0) [inline docs-preview/src/markdown-extensions/video.ts]: Unfortunately the iframe requires the VSCode content security policy to be disabled in order to render the iframe.

Comment (jamarw) [reply docs-preview/src/markdown-extensions/video.ts]: Trying to find out if there's an image we can use for this.

Comment (nschonni) [reply docs-preview/src/markdown-extensions/video.ts]: Maybe something from https://www.youtube.com/yt/about/brand-resources/

Comment (bharney0) [reply docs-preview/src/markdown-extensions/video.ts]: Video preview should look like this now:
image

Unfortunately the video still does not play. But its better than showing the markdown: 
> [!VIDEO https://channel9.msdn.com/Series/Youve-Got-Key-Values-A-Redis-Jump-Start/03/player]

**Pull request merged**